pkgs/by-name/ti/tigerjython/package.nix

```diff
@@ -0,0 +1,88 @@
+# https://nixos.wiki/wiki/Packaging/Binaries https://nixos.wiki/wiki/Java
+{ stdenv
+, lib
+, fetchurl
+, makeWrapper
+, makeDesktopItem
+, copyDesktopItems
+, jre
+}:
+
+let
+  icon = fetchurl {
+    url = "https://raw.githubusercontent.com/tigerjython/tjinstall/master/tjlogo64.png";
+    sha256 = "sha256-tw3uDWLtcMHYmN6JGsEvVKLgI09v5DF27V2+OF9Z5tA=";
+  };
+in
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tigerjython";
+  version = "2.39";
+
+  src = fetchurl {
+    url = "https://tigerjython.ch/user/pages/download/TigerJython.tar.gz";
+    hash = "sha256-rfksd3faLHaXS79PvbQEYLdVaO5/6/4Ytp23amya5zI=";
+  };
```

Comment:
Hey, thanks for taking your time to contribute!

I don't think a package having an unstable download link would be accepted inside nixpkgs. Once the website updates to a later version of the package, that will be fetched instead, leading to some hash mismatch errors.
If there is a way to download specific versions, then great, please do that. Otherwise, I don't know what can be done.